drivers/usb/device/Kconfig

```diff
@@ -53,6 +53,8 @@ config USB_NRF52840
 	depends on SOC_NRF52840
 	select USB_DEVICE_DRIVER
 	select HAS_DTS_USB
+	select NRFX_USBD
+	select NRFX_SYSTICK
```

Comment:
This should be selected by the NRFX_USBD option itself. Just like NRFX_NFCT selects NRFX_TIMER4.